Topic5/Topic5.sql

```diff
@@ -0,0 +1,80 @@
+CREATE DATABASE high_school;
+
+USE high_school;
+
+CREATE TABLE Student(
+	first_name VARCHAR(20) NOT NULL,
+	last_name VARCHAR(20) NOT NULL,
+	registration_number INT UNSIGNED NOT NULL,

```

Comment:
This field could be declared as an autoincremental to avoid the need to define it on an Insert clause